helpers/manifest-parser.js

```diff
@@ -141,8 +141,13 @@ function parseIcon(raw) {
   let sizes = parseString(raw.sizes);
   if (sizes.value !== undefined) {
     let set = new Set();
-    sizes.value.split(/\s/).forEach(size => set.add(size.toLowerCase()));
-
+    sizes.value.split(/\s/).forEach(size => {
+      const pair = size.split(/x/i);
+      if (pair[0] !== pair[1]) {
```

Comment:
my only question on this is if we want this to live in the parser. Just a warning, so it may be fine, but non-square icons are a-ok as far as the spec is concerned.